GUI/electron/src/main.ts

```diff
@@ -243,16 +243,20 @@ ipcMain.on('getGeneratorGUISettings', (event, arg) => {
   };
 
   //Load built in presets
-  let builtInPresetsPath = path.normalize(pythonRootPath + "data/presets_default.json");
+  let builtInPresetsPath = path.normalize(pythonRootPath + "data/Presets");
```

Comment:
As discussed, this should still account for the preset_default.json instead of removing it